plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseTypeMapping.java

```diff
@@ -439,11 +439,44 @@ public void testDecimal()
                 .execute(getQueryRunner(), clickhouseCreateAndInsert("tpch.test_nullable_decimal"));
     }
 
+    private SqlDataTypeTest textAsBinaryRoundTripTest(String inputType)
```

Comment:
Change to static and move to after `testTrinoVarchar`. 

Also, please follow the commit message guideline: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
Done.

Reply:
Renamed commit to `Extra unit tests for type mapping`